package.json

```diff
@@ -32,7 +32,9 @@
     "react-motion": "^0.5.2",
     "react-responsive": "^6.1.1",
     "react-syntax-highlighter": "^10.1.2",
-    "suomifi-ui-components": "0.2.4"
+    "styled-components": "4.3.2",
+    "suomifi-icons": "^0.0.12",
```

Comment:
This should be locked to absolute version to avoid issues when deploying the site.